doc/introduction/interfaces.rst

```diff
@@ -129,8 +129,9 @@ NumPy
 ~~~~~
 
 When using the standard NumPy framework, PennyLane offers some built-in optimizers.
-Some of these are specific to quantum optimization, such as the :class:`~.QNGOptimizer`, :class:`~.LieAlgebraOptimizer`
-:class:`~.RotosolveOptimizer`, :class:`~.RotoselectOptimizer`, :class:`~.ShotAdaptiveOptimizer`, and :class:`~.QNSPSAOptimizer`.
+Some of these are specific to quantum optimization, such as the :class:`~.QNGOptimizer`, :class:`~.LieAlgebraOptimizer`,
```

Comment:
```suggestion
Some of these are specific to quantum optimization, such as the :class:`~.QNGOptimizer`,
```
Since the name is now deprecated, I would remove it from this list.

Reply:
We should keep the entry though and use the new name

Reply:
I've removed the old name from this paragraph and the list right below. Thanks! :)